### ISSUE TITLE
Always update tests/elm-package.json

### ISSUE THREAD

**Issue by boxed** [rtfeldman/node-test-runner#210]
It would be nice if the "--add-dependencies" command was always run when doing "elm-test". Then point 4 from "Running tests locally" at https://github.com/elm-community/elm-test could could be removed and it would make the development experience smoother.

Alternatively the runner could error out if you haven't updated the dependencies and tell you to run the add dependencies command.

**Comment by boxed** [comment]
I’m confused. Did you just close this issue because a command was renamed?

**Comment by rtfeldman** [comment]
No. `--add-dependencies` is gone because it wouldn't have any effect in the Elm 0.19 version of `elm-test`, so the request to run it every time `elm-test` gets run wouldn't make any sense either.

**Comment by boxed** [comment]
But then you wrote it has "been replaced with elm-test install"? So will elm-test install do what "add-dependencies" did before? If so then this issue should just have the title and description updated to match the new reality, not just being closed.

**Comment by boxed** [comment]
@zwilias Thanks. Yes, that explains it. Thanks.
